cap_package/CAP/Code/cdi_validator.py

```diff
@@ -21,7 +21,7 @@ def CDI_Masterlist_QA(cdi_dataset):
 
 	# Crossreference the CKAN API for dataset and check/update masterlist values
 	name_change, catalog_change = Check_Name_and_Update_Caturl(cdi_dataset, api_json)
-	catalog_change = Check_Catalog_URL_Status(cdi_dataset, catalog_change)
+	#catalog_change = Check_Catalog_URL_Status(cdi_dataset, catalog_change)
```

Comment:
once you confirm you no longer need the commented out code, it's good to delete them!

Reply:
Will do! These are some changed to the cap package that will be updated soon